api/tokens.go

```diff
@@ -0,0 +1,179 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/ONSdigital/dp-identity-api/emailvalidation"
+
+	"github.com/ONSdigital/log.go/log"
+)
+
+type ErrorStructure struct {
+	Errors []IndividualError `json:"errors"`
+}
+
+type IndividualError struct {
+	SpecificError string `json:"error"`
+	Message       string `json:"message"`
+	Source        Source `json:"source"`
+}
+
+type Source struct {
+	Field string `json:"field"`
+	Param string `json:"param"`
+}
+
+func TokensHandler() http.HandlerFunc {
+
+	return func(w http.ResponseWriter, req *http.Request) {
+
+		ctx := req.Context()
+
+		field := ""
+		param := ""
```

Comment:
This is absolutely fine but I think the Go convention is:

`var field string`

When you declare a var this way it will have the _zero value_ for that type which for String will be "".  As I said nothing technically wrong with this.

Reply:
Thank you for the suggestion